virusscanner/terraform/main.tf

```diff
@@ -0,0 +1,96 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "4.52.0"
+    }
+  }
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = var.region
+}
+
+resource "aws_vpc" "virus_scanning_vpc" {
+  cidr_block = "10.0.0.0/16"
+  tags = {
+    Name = "Virus scanning VPC"
+  }
+}
+
+resource "aws_subnet" "virus_scanning_subnet1" {
+  availability_zone = "eu-west-2a"
+  vpc_id            = aws_vpc.virus_scanning_vpc.id
+  cidr_block        = "10.0.1.0/24"
+
+  tags = {
+    Name = "Virus scanning subnet for eu-west-2a"
+  }
+}
+
+resource "aws_subnet" "virus_scanning_subnet2" {
+  availability_zone = "eu-west-2b"
+  vpc_id            = aws_vpc.virus_scanning_vpc.id
+  cidr_block        = "10.0.2.0/24"
+
+  tags = {
+    Name = "Virus scanning subnet for eu-west-2b"
+  }
+}
```

Comment:
Is it possible to use the vpc module here? Also, Should we re use the ARF virus scanner VPC for now to save on resources?

Reply:
I don't think so, as this is kept in it's own Terraform folder and does not have access to the tfvars files in the general infrastructure terraform folder

Additionally, the ARF virus scanner is no longer in use; anything uploaded to the ARF project goes through the fake virus scanner only and is automatically marked as Clean

Reply:
The virus scanner is not in use but the vpc is still up.
Yes you're right, I didn't realise this is a new folder outside of the main one.